map/src/data/DOCUMENTATION.md

```diff
@@ -0,0 +1,110 @@
+# Map Marker Documentation
+
+The purpose of this readme is to outline the data model that the map uses to track organization/community map markers.
+
+## What is a Map Marker
+
+A map marker represents an organization or community effort to orchestrate aid. It consists of several components as well as a latitude, longitude, and area of influence (a radius measured in meters).
+
+### Data Model
+
+```typescript
```

Comment:
is it worth duplicating this here? perhaps we just add a bit more tsdoc and then link to the relevant lines in the ts? Copying this here leaves potential for it becoming outdated.

Reply:
Yes 😞 

I think we can open an issue to run ts doc and to replace this. Once hosting is up we can actually just host all the docs for the map... on the map!

Reply:
I'll merge this and PR again to rename.

Reply:
Hosting is up, we could add this now, though to be honest if someone's going to be editing the same file anyway, i don't think that generating docs from the tsdocs and hosting them anywhere really adds anything?